### PR TITLE
[Merged by Bors] - chore: fix spelling mistakes in category theory files

### PR DESCRIPTION
**Pull request by euprunin**
---

### CODE REVIEW

**Review by dagurtomas — APPROVED** [review]
You need to split up a line that's too long in `CoherentSheaves.lean`, otherwise LGTM

File: Mathlib/CategoryTheory/GradedObject/Trifunctor.lean [hunk 1]
```diff
@@ -268,7 +268,7 @@ variable {r} (ρ₁₂ : BifunctorComp₁₂IndexData r)
 /-- Given bifunctors `F₁₂ : C₁ ⥤ C₂ ⥤ C₁₂`, `G : C₁₂ ⥤ C₃ ⥤ C₄`, graded objects
 `X₁ : GradedObject I₁ C₁`, `X₂ : GradedObject I₂ C₂`, `X₃ : GradedObject I₃ C₃` and
 `ρ₁₂ : BifunctorComp₁₂IndexData r`, this asserts that for all `i₁₂ : ρ₁₂.I₁₂` and `i₃ : I₃`,
-the functor `G(-, X₃ i₃)` commutes wich the coproducts of the `F₁₂(X₁ i₁, X₂ i₂)`
+the functor `G(-, X₃ i₃)` commutes which the coproducts of the `F₁₂(X₁ i₁, X₂ i₂)`
```
Comment (dagurtomas) [inline Mathlib/CategoryTheory/GradedObject/Trifunctor.lean:271]:
```suggestion
the functor `G(-, X₃ i₃)` commutes with the coproducts of the `F₁₂(X₁ i₁, X₂ i₂)`
```

File: Mathlib/CategoryTheory/GradedObject/Trifunctor.lean [hunk 2]
```diff
@@ -448,7 +448,7 @@ variable {r} (ρ₂₃ : BifunctorComp₂₃IndexData r)
 /-- Given bifunctors `F : C₁ ⥤ C₂₃ ⥤ C₄`, `G₂₃ : C₂ ⥤ C₃ ⥤ C₂₃`, graded objects
 `X₁ : GradedObject I₁ C₁`, `X₂ : GradedObject I₂ C₂`, `X₃ : GradedObject I₃ C₃` and
 `ρ₂₃ : BifunctorComp₂₃IndexData r`, this asserts that for all `i₁ : I₁` and `i₂₃ : ρ₂₃.I₂₃`,
-the functor `F(X₁ i₁, _)` commutes wich the coproducts of the `G₂₃(X₂ i₂, X₃ i₃)`
+the functor `F(X₁ i₁, _)` commutes which the coproducts of the `G₂₃(X₂ i₂, X₃ i₃)`
```
Comment (dagurtomas) [inline Mathlib/CategoryTheory/GradedObject/Trifunctor.lean:451]:
```suggestion
the functor `F(X₁ i₁, _)` commutes with the coproducts of the `G₂₃(X₂ i₂, X₃ i₃)`
```

**Comment by dagurtomas** [comment]
Sorry, didn't catch these two errors earlier. Everything else is fine

**Comment by dagurtomas** [comment]
You can just remove the `awaiting author` label when you've addressed all comments from a review, this will make the PR show up again on the review queue

**Review by jcommelin — COMMENTED** [review]
Thanks :tada:

bors merge

**Pull request closed without merging**